### PR TITLE
For analyses with the IDR samples produced in 2018.

### PR DESCRIPTION
**Pull request by ryonamin**
BEGINRELEASENOTES
- Add example steering files and necessary data files for analyses (6jet, 500GeV, w/ beam bkg.) with the IDR samples produced in 2018.
ENDRELEASENOTES

### CODE REVIEW

File: LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_2Jets_NewVtx.xml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+<global>
+	<parameter name="LCIOInputFiles"> 
+__INPUTFILES__
+        </parameter>
+	<parameter name="MaxRecordNumber" value="0" />  
+	<parameter name="SkipNEvents" value="0" />  
+	<parameter name="SupressCheck" value="false" />  
+	<parameter name="GearXMLFile"> 
+__GEARFILE__
```
Comment (rete) [inline LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_2Jets_NewVtx.xml]: Why do you have to setup a Gear file ? Gear has been deprecated for the IDR production. You normally don't need it

File: LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_4Jets_NewVtx.xml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+<global>
+	<parameter name="LCIOInputFiles"> 
+__INPUTFILES__
+        </parameter>
+	<parameter name="MaxRecordNumber" value="0" />  
+	<parameter name="SkipNEvents" value="0" />  
+	<parameter name="SupressCheck" value="false" />  
+	<parameter name="GearXMLFile"> 
+__GEARFILE__
```
Comment (rete) [inline LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_4Jets_NewVtx.xml]: Same here

File: LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_6Jets_NewVtx.xml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+<global>
+	<parameter name="LCIOInputFiles"> 
+__INPUTFILES__
+        </parameter>
+	<parameter name="MaxRecordNumber" value="0" />  
+	<parameter name="SkipNEvents" value="0" />  
+	<parameter name="SupressCheck" value="false" />  
+	<parameter name="GearXMLFile"> 
+__GEARFILE__
```
Comment (rete) [inline LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_6Jets_NewVtx.xml]: Same here

File: LCFIPlusConfig/steer/v04/revertexing_500GeV.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+	<processor name="VertexFinder"/>
+	<processor name="MyLCIOOutputProcessor"/>
+</execute>
+
+<global>
+	<parameter name="LCIOInputFiles"> 
+__INPUTFILES__
+        </parameter>
+	<parameter name="GearXMLFile"> 
+__GEARFILE__
```
Comment (rete) [inline LCFIPlusConfig/steer/v04/revertexing_500GeV.xml]: Same here

File: LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_2Jets_NewVtx.xml (hunk 1 above)
Comment (ryonamin) [reply LCFIPlusConfig/steer/v04/jetclusteringAndFlavourtagging_2Jets_NewVtx.xml]: Thank you for spotting this part. I didn't pay attention for that part but I confirmed that indeed the magnetic field has been taken care of throught MarlinUtil. So I remove those parts.

**Pull request merged**